### PR TITLE
Create landing page cards

### PR DESCRIPTION
**Pull request by OptimumCOW**
https://trello.com/c/5xIWeS4S/13-add-card-components-to-landing-page-for-rovers

### CODE REVIEW

File: src/styles/Home.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
+@import "./constants.scss";
+
+#Home {
```
Comment (oskarwilliams) [inline src/styles/Home.scss]: could you convert these to classes

**Review by oskarwilliams — CHANGES_REQUESTED** [review]

File: src/styles/Home.scss (hunk 1 above)
Comment (OptimumCOW) [reply src/styles/Home.scss]: This has been addressed.

**Comment by OptimumCOW** [comment]
Updated.

**Comment by OptimumCOW** [comment]
I made some changes to styling the home page banner and altered the way the card provides the image padding to enable the cards to line up to the banner image nicely. All checks passed.

**Review by oskarwilliams — APPROVED** [review]

**Pull request merged**